apps/src/netsim/NetSimTable.js

```diff
@@ -133,11 +172,11 @@ NetSimTable.prototype.readAll = function (callback) {
  */
 NetSimTable.prototype.makeThrottledRefresh_ = function (waitMs) {
   return _.throttle(function () {
-    this.clientApi_.all(function (err, data) {
-      if (err === null) {
-        this.fullCacheUpdate_(data);
-      }
-    }.bind(this));
+    if (this.useIncrementalRefresh_) {
```

Comment:
I've replaced both `refreshAll` and `refreshFromRowID` with a `refresh` function that behaves differently depending on how the table is configured.  This was a fairly invasive change, because we were still calling `refreshAll` in many places (especially in tests), and those clients now need to use a combination of `refresh()` and `readAll()` if they want equivalent behavior.

@philbogle please take another look.